pkgs/os-specific/linux/kbd/default.nix

```
@@ -1,15 +1,14 @@
-{ lib, stdenv, fetchurl, autoreconfHook,
-  gzip, bzip2, pkg-config, flex, check,
-  pam, coreutils
+{ lib, stdenv, fetchurl, nixosTests, autoreconfHook,
+  pkg-config, flex, check, pam, coreutils
```

I don't think we have specified this in the documentation, but I believe the inofficially™ agreed upon style is more like this:

```suggestion
{ lib, stdenv, fetchurl, nixosTests, autoreconfHook
, pkg-config, flex, check, pam, coreutils
```

---

i think the closest we have to a code style convention is nixpkgs-fmt , so i used it (for the first time) to format it